components/Login/SignIn.tsx

```diff
@@ -0,0 +1,105 @@
+import React from 'react'
+import { Form, Formik, Field } from 'formik'
+import { StyledTextInput } from '../FormStyles/InputBox'
+import * as Yup from 'yup'
+import {
+  BackButton,
+  EStyledButton,
+  Container,
+  SubContainer,
+  Title,
+  SubTitle,
+} from '../Login/SignUp'
+import styled from 'styled-components'
+import Link from 'next/link'
+import { useRouter } from 'next/router'
+import VisibilityIcon from '@mui/icons-material/Visibility'
+import VisibilityOffIcon from '@mui/icons-material/VisibilityOff'
+import { PasswordInputBox } from '../FormStyles/PasswordInputBox'
+
+export const RememberSignIn = styled.div`
+  display: flex;
+  justify-content: flex-end;
+  align-items: center;
+  gap: 25px;
+  width: 100%;
+`
+
+export const RememberDiv = styled.div`
+  margin-top: 2%;
+`
+
+interface FormValues {
+  email: string
+  password: string
+  checked: Array<string>
+}
+
+function SignIn() {
+  const router = useRouter()
+  const initialVal: FormValues = {
+    email: '',
+    password: '',
+    checked: [],
+  }
+
+  return (
+    <Formik
+      initialValues={initialVal}
+      validationSchema={Yup.object({
+        email: Yup.string().email('Invalid email address').required('Required'),
+        password: Yup.string().required('Required'),
+      })}
+      onSubmit={(values, { setSubmitting }) => {
+        setTimeout(() => {
+          alert(JSON.stringify(values, null, 2))
+          setSubmitting(false)
+        }, 400)
+      }}
+    >
+      <Form>
+        <Container>
+          <Link href="/home" passHref>
+            <BackButton type="button">&lt; Back</BackButton>
+          </Link>
+          <Title>Welcome back!</Title>
+          <SubTitle>Please sign in.</SubTitle>
+          <SubContainer>
+            <StyledTextInput
+              width="600px"
+              height="53px"
+              name="email"
+              placeholder="Email"
+              type="text"
+            />
+
+            <PasswordInputBox
+              width="600px"
+              height="53px"
+              placeholder="Password"
+              name="password"
+            />
```

Comment:
NIT: since there's a couple of places with consistent styling on components like this, it might be worth having a file for stylized components (ex have a component just for a passwordbox with set width and height so if we change it once it changes everywhere or a StylizedTextInput that's only used for Emails since it's used in 3-4 places)